src/main/java/frc/robot/subsystems/Intake/IntakeIOSim.java

```diff
@@ -0,0 +1,66 @@
+package frc.robot.subsystems.Intake;
+
+
+import static frc.robot.GlobalConstants.SIM_DELTA_TIME;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import edu.wpi.first.math.system.plant.LinearSystemId;
+
+public class IntakeIOSim implements IntakeIO {
+    private SingleJointedArmSim pivotSim;
+    private DCMotorSim wheelMotorSim;
+    private PIDController pivotController;
+    private PIDController wheelSpeedController;
+
```

Comment:
If you don't want to add ctre sim with the wpilib physics sim open an issue about it so sum1 else can do it once this is merged